tests/checkBoxes.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { test, expect } from "@playwright/test";
+
+test.beforeEach(async ({ page }) => {
+  await page.goto("/");
+
+  await page.getByText(" Veterinarians").click();
+  await page.locator(".dropdown-menu").getByText(" All").click();
+});
+
+test("Validate selected specialties", async ({ page }) => {
+  await expect(page.locator("h2")).toHaveText("Veterinarians");
+
+  const veterinarianHelen = page.locator("tr", {
+    has: page.getByText(" Helen Leary "),
+  });
+  await veterinarianHelen.getByRole("button", { name: "Edit Vet" }).click();
```

Comment:
`veterinarianHelen` constant used just once so lines 13 and 16 can be combined in a single line click action